### PR TITLE
About us

### PR DESCRIPTION
**Pull request by Shubham-Kumar-2000**
## Issue that this pull request solves

 Closes: #482 

## Proposed changes

About Us page should have two cards similar to faqs

Manage Teams
Add Team Member
Manage Teams will take us to a new page having cards with dummy data as of now.

Add Team Member card will take us to a page containing form with following fields along with a submit button -

Full Name
picture Upload
Short Description
linkedIn URL
Github URL
Twitter URL
Teams(Open Source, Broadcast, Design, Social, Resource Sharing) =>This will be a multiple select box with the options given
Please make sure to render the page on the right side keeping the left side navigation panel same for these pages.

### Brief description of what is fixed or changed

All Above mentioned changes have been done

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

image
image
image
image

## Other information

NA

### CODE REVIEW

File: frontend/src/pages/Admin/Components/About/About.jsx [hunk 1]
```diff
@@ -1,9 +1,41 @@
 import React from "react";
+import style from "./about-us.module.scss";
 
-export function About() {
+export function About(props) {
+  const cards = [
+    {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/About/About.jsx:6]: Hey @Shubham-Kumar-2000 Please make these cards look like :arrow_down: 
Screenshot from 2021-03-12 13-43-26

File: frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import style from "./manage-teams.module.scss";
+
+export function ManageTeams(props) {
+  const cards = [
+    {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/MangeTeams/ManageTeams.jsx:6]: let's put these data under `test_data/teams.json` [All testing data are kept there] Pleasenote that we don't have a file named `teams.json` so please create it under `test_data` folder

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @Shubham-Kumar-2000 This looks super nice and clean Btw the issue was only for `add_team` but as you have created `manage teams` as well so I am changing the level of the issue :)

Please make the requested changes so that we can merge this.

File: frontend/src/pages/Admin/Components/AddTeamMember/AddTeamMember.jsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+                <input
+                  id="txt_twitter"
+                  type="url"
+                  required="required"
+                  name="twitter"
+                  placeholder="Twitter URL"
+                />
+                <i className="fab fa-twitter"></i>
+              </div>
+              <label className={styles["add-team-member-label-text"]}>Teams :</label>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/AddTeamMember/AddTeamMember.jsx:119]: Let's keep it's color black/grey(same as input fields place-holder) In order to be cosistent

**Comment by Shubham-Kumar-2000** [comment]
All the required changes made.

@Kajol-Kumari

**Review by Kajol-Kumari — APPROVED** [review]
Nice and Clean! LGTM :+1: 
Thanks for incorporating the suggested changes @Shubham-Kumar-2000

**Pull request merged**